src/queries/petType/editPetTypeQuery.ts

```diff
@@ -0,0 +1,17 @@
+import PetType from '../../models/PetType'
+import { type IPetType } from '../../interfaces/models'
+
+const editPetTypeQuery = async (typeId: number, title: string): Promise<IPetType | null> => {
+  const type = await PetType.findByPk(typeId)
+
+  if (type != null) {
+    type.title = title
+    await type.save()
+
+    return type
+  }
+
+  return null
+}
+
+export default editPetTypeQuery
```

Comment:
use this, please 
https://sequelize.org/docs/v6/core-concepts/model-querying-basics/#simple-update-queries

```suggestion
import PetType from '../../models/PetType'
import { type IPetType } from '../../interfaces/models'

const editPetTypeQuery = async (typeId: number, title: string):
Promise<[affectedCount: number, affectedRows: IPetType[]]> => {
  const result = await PetType.update({ title }, { where: { typeId }, returning: true })

  return result
}

export default editPetTypeQuery
```